### DISCUSSION TITLE
A net for Xiangqi would be great

### DISCUSSION THREAD

**Discussion by ghost** [fairy-stockfish/Fairy-Stockfish#314] [Q&A]
Is it possible to create a Xiangqi net?
Is it possible to make Fairy-Stockfish faster for Xiangqi? It is extreme slow on my hardware.

**Comment by ianfab** [comment]
> Is it possible to create a Xiangqi net?

In principle yes, and that is an important goal. However, this requires some more generalization of the NNUE architecture in Fairy-Stockfish, and due to other open tasks progress is slow, but I am working on it.

> Is it possible to make Fairy-Stockfish faster for Xiangqi? It is extreme slow on my hardware.

It should be possible to reduce the overhead for generalization to some extent, also see https://github.com/ianfab/Fairy-Stockfish/issues/253. Contributions on this are very welcome.
